docs/windows.md

```diff
@@ -1,142 +1,141 @@
+
+### `gx`
+You may install prebuilt binaries for [`gx`](https://dist.ipfs.io/#gx) & [`gx-go`](https://dist.ipfs.io/#gx-go) if they're available for your platform.
+Alternatively you can build them from source and install them to `%GOPATH%\bin` by running the following:
 
 ```
-go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=XXXXXXX"
+go get -u github.com/whyrusleeping/gx
+go get -u github.com/whyrusleeping/gx-go
 ```
 
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
-
-You can check that the ipfs you built has the right version using:
+### `go-ipfs`
 
 ```
-ipfs version --commit
+SET PATH=%PATH%;%GOPATH%\bin
+go get -u -d github.com/ipfs/go-ipfs
+cd %GOPATH%/src/github.com/ipfs/go-ipfs
+gx --verbose install --global
+cd cmd\ipfs
 ```
+We need the `git` commit hash to be included in our build so that in the extremely rare event a bug is found, we have a refference point later for tracking them. We'll ask `git` for it and store it in a variable. Unfortunatley the syntax for the next command is different depending on how it's executed. If you're on the interactive command line vs executing from inside the batch/cmd interpriter. Use the one that applies to you.  
+interactive: `FOR /F %V IN ('git rev-parse --short HEAD') do set SHA=%V`  
+interpreter: `FOR /F %%V IN ('git rev-parse --short HEAD') do set SHA=%%V`  
 
-It should output something like "ipfs version 0.4.0-dev-XXXXXXX" where
-XXXXXXX is the current commit that you passed to the build command.
-
-## Cygwin way: install Cygwin
-
-Install Cygwin as explained in the Cygwin documentation:
-
-http://cygwin.com/install.html
-
-By default Cygwin will not install 'make', so you should click on the
-"Devel" category during the Cygwin installation process and then check
-the 'make' package.
-
-## Cygwin way: build go-ipfs
-
-To build go-ipfs using Cygwin you just need to open a Cygwin Terminal
-and then type the following commands:
-
+Finally, we'll build and test `ipfs` itself.
 ```
-cd $GOPATH/src/github.com/ipfs/go-ipfs
-make install
+go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=%SHA%"
+%GOPATH%\bin\ipfs.exe version --all
 ```
+You can check that the ipfs output versions match with `go version` and `git rev-parse --short HEAD`, if `ipfs.exe` executes and everything matches, then building was successful.
 
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
+## Troubleshooting
+- Git auth  
+If you get authentication problems with Git, you might want to take a look at https://help.github.com/articles/caching-your-github-password-in-git/ and use the suggested solution:  
+`git config --global credential.helper wincred`
 
-You can check that the ipfs you built has the right version using:
+- Anything else  
+Please search <https://discuss.ipfs.io/> for any additional issues you encounter, if you can't find an existing resolution, feel free to make a post asking for help.
```

Comment:
"Please search <https://discuss.ipfs.io/search?q=windows%20category%3A13> for any additional issues you may encounter. If you can't find any existing resolution, feel free to post a question asking for help."

Of course, if they are behind the great firewall, this service will not be available - but then again neither will github...